services-core/src/gas_price/gnosis_safe.rs

```diff
@@ -24,22 +25,32 @@ pub fn api_url_from_network_id(network_id: &str) -> Option<&'static str> {
 
 /// Gas prices in wei retrieved from the gas station. This is a result from the
 /// API call.
-#[derive(Deserialize, Debug, Default, Eq, PartialEq)]
+#[derive(Deserialize, Debug, Default)]
 #[serde(rename_all = "camelCase")]
 pub struct GasPrices {
     pub last_update: String,
-    #[serde(deserialize_with = "deserialize_u256_from_string")]
-    pub lowest: U256,
-    #[serde(deserialize_with = "deserialize_u256_from_string")]
-    pub safe_low: U256,
-    #[serde(deserialize_with = "deserialize_u256_from_string")]
-    pub standard: U256,
-    #[serde(deserialize_with = "deserialize_u256_from_string")]
-    pub fast: U256,
-    #[serde(deserialize_with = "deserialize_u256_from_string")]
-    pub fastest: U256,
+    #[serde(with = "display_fromstr")]
+    pub lowest: f64,
```

Comment:
I changed this to f64 because it is easier to work with and we don't lose any precision because numbers aren't that high and we are converting to float anyway.

Reply:
Makes sense. Also, I think for the kinds of calculations you are doing, `U256` is not well suited.